### PR TITLE
React order entry 002

### PR DESCRIPTION
**Pull request by abertnamanya**
- updates to the Order Entity React UI

### CODE REVIEW

**Comment by abertnamanya** [comment]
Thanks @CalebSLane

File: react-ui/src/components/addOrder/OrderReferralRequest.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                    })
+                }
+            </Select>
+        </>);
+    };
+
+    const SentDate = ({index}) => {
+        return (<>
+            <DatePicker dateFormat="m/d/Y" datePickerType="single">
+                <DatePickerInput id={`sendDate_` + index} placeholder="dd/mm/yyyy" type="text" labelText=""/>
```
Comment (CalebSLane) [inline react-ui/src/components/addOrder/OrderReferralRequest.js:93]: we will need the date format to be configurable based on the Site Information

File: react-ui/src/components/addOrder/OrderReferralRequest.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    TableCell,
+    TableHead,
+    TableHeader,
+    TableRow,
+    TextInput
+} from "@carbon/react";
+import {UserInformationContext} from "../layout/Layout";
+
+
+const header = [{key: 'reason', header: 'Reason for Referral*'}, {key: 'referrer', header: 'Referrer'}, {
```
Comment (CalebSLane) [inline react-ui/src/components/addOrder/OrderReferralRequest.js:20]: headers will need to be message keys for translation

**Review by CalebSLane — APPROVED** [review]
Few notes around internationalization, but are not necessary to merge in the code.

File: react-ui/src/components/addOrder/OrderReferralRequest.js (hunk 2 above)
Comment (abertnamanya) [reply react-ui/src/components/addOrder/OrderReferralRequest.js:20]: Oh yeah, Let me start mapping the text to keys for translation.

File: react-ui/src/components/addOrder/OrderReferralRequest.js (hunk 1 above)
Comment (abertnamanya) [reply react-ui/src/components/addOrder/OrderReferralRequest.js:93]: Do we have an endpoint that queries site configurations?

**Pull request merged**